Service/src/main/java/Senders/EmailSender.java

```diff
@@ -0,0 +1,24 @@
+package Senders;
+
+import TIcket.Ticket;
+import org.example.SetClassId;
+
+import java.time.format.DateTimeFormatter;
+
+public class EmailSender extends SetClassId implements TicketSender {
+    private static int classId;
```

Comment:
What is the reason of making the field static?

Reply:
The ID must relate directly to the class, and not to the object of this class